autoware_launch/config/system/mrm_emergency_stop_operator/mrm_emergency_stop_operator.param.yaml

```diff
@@ -1,5 +1,5 @@
 /**:
   ros__parameters:
     update_rate: 30
-    target_acceleration: -2.5
-    target_jerk: -1.5
+    target_acceleration: -3.0
```

Comment:
Is it OK that the acceleration is inconsistent with the emergency stop in vehicle_cmd_gate?
https://github.com/autowarefoundation/autoware.universe/blob/aab396a342ae5f1e8ae433cf4b0b51359359957d/control/autoware_vehicle_cmd_gate/config/vehicle_cmd_gate.param.yaml#L13

Reply:
@isamu-takagi  thanks for your review and question.
The "emergency_acceleration" specified on the vehicle_cmd_gate is used when the emergency heartbeat times out and when there is an external emergency command as you can see [here](https://github.com/autowarefoundation/autoware.universe/blob/4e9755b2f8f18221be6d5840a33addc5f10e5d23/control/autoware_vehicle_cmd_gate/src/vehicle_cmd_gate.cpp#L400C5-L406C4) and [here](https://github.com/autowarefoundation/autoware.universe/blob/4e9755b2f8f18221be6d5840a33addc5f10e5d23/control/autoware_vehicle_cmd_gate/src/vehicle_cmd_gate.cpp#L421C5-L430C4):  These 2 are the only places where publishEmergencyStopControlCommands() is called, and that function is the one that calls [createEmergencyStopControlCmd()](https://github.com/autowarefoundation/autoware.universe/blob/4e9755b2f8f18221be6d5840a33addc5f10e5d23/control/autoware_vehicle_cmd_gate/src/vehicle_cmd_gate.cpp#L674) which is the only place where the vehicle_cmd_gate's "emergency_acceleration"  is used as far as I can tell.

So, I believe it is not necessary to change this specific value of "emergency_acceleration" for cmd_gate to match the mrm_emergency_stop values.